ci/run_cpp_benchmark_smoketests.sh

```diff
@@ -14,12 +14,20 @@ export OMPI_ALLOW_RUN_AS_ROOT=1  # CI runs as root
 export OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
 export OMPI_MCA_opal_cuda_support=1  # enable CUDA support in OpenMPI
 
+ls -lh .
+
 # Ensure that benchmarks are runnable
 python "${TIMEOUT_TOOL_PATH}" 30 \
     mpirun --map-by node --bind-to none -np 3 ./bench_shuffle -m cuda
+
 python "${TIMEOUT_TOOL_PATH}" 30 \
     mpirun --map-by node --bind-to none -np 3 ./bench_comm -m cuda
-./bench_streaming_shuffle -m cuda
+
+python "${TIMEOUT_TOOL_PATH}" 30 \
+  ./bench_memory_resources --benchmark_min_time=0s
```

Comment:
What does `--benchmark_min_time=0s` do? Feels like what the default would be, no?